cmd/fyne/internal/mobile/build.go

```diff
@@ -273,10 +273,14 @@ var (
 )
 
 // RunNewBuild executes a new mobile build for the specified configuration
-func RunNewBuild(target, appID, icon, name, version string, build int, release bool, cert, profile string) error {
+func RunNewBuild(target, appID, icon, name, version string, build int, release, distribution bool, cert, profile string) error {
 	buildTarget = target
 	buildBundleID = appID
-	buildRelease = release
+	buildRelease = distribution
+	if release {
+		buildLdflags = "-w"
```

Comment:
Should this not have `-s` as well? The release binaries usually use `-ldflags="-s -w"` but I am not sure why this only has `-w`.

Reply:
> Should this not have `-s` as well? The release binaries usually use `-ldflags="-s -w"` but I am not sure why this only has `-w`.

On android the application is a .so linked dynamically with the jvm. -s remove the symbol which will break the dynamic linking.  -w remove the dwarf symbol, also known as debugging symbol. This are safe to remove as only user by debugger.

Reply:
Ah, I see. Sounds sensible